src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6009,7 +6059,7 @@ namespace SimAirServingZones {
 
 			// write predefined standard 62.1 report data
 			for ( AirLoopNum = 1; AirLoopNum <= NumPrimaryAirSys; ++AirLoopNum ) {
-				if ( FinalSysSizing( AirLoopNum ).SystemOAMethod == SOAM_VRP ) {
+//				if ( FinalSysSizing( AirLoopNum ).SystemOAMethod == SOAM_VRP ) {  // commented line allows ZoneSum method to report tables to the html file
```

Comment:
@rraustad It's an interesting idea to allow the ZoneSum method to populate the Std 62 report.  Is this intended just for debugging?  If it's intended to stay, then the report needs to indicate which method was used.